### PR TITLE
Add flags so that no prompting is possible

### PR DESCRIPTION
**Pull request by holisticode**

### CODE REVIEW

File: cmd/subnetcmd/join.go [hunk 1]
```diff
@@ -58,7 +64,10 @@ This command currently only supports subnets deployed on the Fuji testnet.`,
 	}
 	cmd.Flags().StringVar(&avagoConfigPath, "avalanchego-config", "", "file path of the avalanchego config file")
 	cmd.Flags().StringVar(&pluginDir, "plugin-dir", "", "file path of avalanchego's plugin directory")
+	cmd.Flags().StringVar(&networkStr, "network", "", "avalanche network where the validator will be joining (either `fuji` or `mainnet`)")
```
Comment (dasconnor) [inline cmd/subnetcmd/join.go:67]: Same global decision I mentioned in the other PR

File: cmd/subnetcmd/join.go [hunk 2]
```diff
@@ -133,7 +153,10 @@ but until the node is whitelisted, it will not be able to validate this subnet.`
 		return nil
 	}
 
+	// if **both** flags were set, nothing special needs to be done
```
Comment (dasconnor) [inline cmd/subnetcmd/join.go:156]: oof, these comments are showing me how confusing this is. We don't need to fix now, but we probably want to think about refactoring in the future to make simpler/more clear.

**Review by dasconnor — COMMENTED** [review]
I think I can approve this after we finalize the decision around network flags

File: cmd/subnetcmd/join.go (hunk 1 above)
Comment (holisticode) [reply cmd/subnetcmd/join.go]: Fixed.

**Review by dasconnor — APPROVED** [review]

**Pull request merged**